snapshot/git/gitdb/setup.go

```diff
@@ -0,0 +1,54 @@
+package gitdb
+
+import (
+	"github.com/scootdev/scoot/ice"
+	"github.com/scootdev/scoot/os/temp"
+	snap "github.com/scootdev/scoot/snapshot"
+	"github.com/scootdev/scoot/snapshot/bundlestore"
+	"github.com/scootdev/scoot/snapshot/git/repo"
+)
+
+type module struct{}
+
+// Module returns a module that supports typical GitDB usage
+func Module() ice.Module {
+	return module{}
+}
+
+// Install installs setup functions to use GitDB
+func (m module) Install(b *ice.MagicBag) {
+	b.PutMany(
+		func(tmp *temp.TempDir) RepoIniter {
+			return &NewRepoIniter{tmp: tmp}
+		},
+		func(store bundlestore.Store) *BundlestoreConfig {
+			return &BundlestoreConfig{Store: store}
+		},
+		func() *StreamConfig {
+			return nil
+		},
+		func() *TagsConfig {
+			return nil
+		},
+		func() AutoUploadDest {
+			return AutoUploadBundlestore
+		},
+		MakeDBNewRepo,
```

Comment:
That may be true now, but I'm not sure it always will be.

And no, I think not necessarily defaults for everything...

I'd rather not try to overfit on too little data rn.